packages/cli/package.json

```diff
@@ -42,7 +42,7 @@
   },
   "aiKey": "1c56be97-bb74-42cf-b04b-8f1aabf04592",
   "devDependencies": {
-    "@azure/arm-subscriptions": "^3.0.0",
+    "@azure/arm-subscriptions": "^5.0.0",
```

Comment:
Remove this since it's already in `dependencies`?

Reply:
Done.